pidgin/errors.py

```diff
@@ -4,6 +4,12 @@ def __init__(self, message):
         self.code = 500
 
 
+class AuthenticationException(PidginException):
+    def __init__(self, message):
+        self.message = str(message)
+        self.code = 511
```

Comment:
I'm actually not sure what a 511 is for, but I would expect the code for this would be either 401 or 403